docs/BINDINGS.md

```diff
@@ -3,7 +3,7 @@ libcurl bindings
 
  Creative people have written bindings or interfaces for various environments
  and programming languages. Using one of these allows you to take advantage of
- curl powers from within your favourite language or system.
+ curl powers from within your favorite language or system.
```

Comment:
I'm not sure we have a set standard for whether to use British or American english in documentation, so I think we should leave this one as is for now.